### ISSUE TITLE
Static content not found if launching application within Eclipse

### ISSUE THREAD

**Issue by gitck** [sbrannen/spring-events#6]
Application should extend `WebMvcAutoConfiguration`; otherwise, _static_ content can not be served when started via 'Run As - Java Application'. (see patch below)

```diff
diff --git a/src/main/java/com/sambrannen/spring/events/Application.java b/src/main/java/com/sambrannen/spring/events/Application.java
index 2bc47b0..3cb757f 100644
--- a/src/main/java/com/sambrannen/spring/events/Application.java
+++ b/src/main/java/com/sambrannen/spring/events/Application.java
@@ -19,6 +19,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.web.WebMvcAutoConfiguration;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 
@@ -34,7 +35,7 @@
  */
 @SpringBootApplication(scanBasePackageClasses = { EventService.class, EventsController.class })
 @EnableGlobalMethodSecurity(prePostEnabled = true)
-public class Application {
+public class Application extends WebMvcAutoConfiguration {
 
 	public static void main(String[] args) {
 		SpringApplication.run(Application.class, args);
```

**Comment by sbrannen** [comment]
Great catch! 👍 

I'll make the change....

Thanks!

**Comment by sbrannen** [comment]
Out of curiosity, how did you determine that extending `WebMvcAutoConfiguration` would solve the problem within the IDE?

**Comment by sbrannen** [comment]
(and.... I apologize for not responding earlier to this issue, but the notification apparently got marked as _spam_ which prevented me from seeing the email).

**Comment by sbrannen** [comment]
Addressed in `master` in commit 939568e3e6493856abb35b39c8af8b5c8768e447.
